src/Module.ts

```diff
@@ -2,7 +2,7 @@ import * as acorn from 'acorn';
 import * as ESTree from 'estree';
 import { locate } from 'locate-character';
 import MagicString from 'magic-string';
-import extractAssignedNames from 'rollup-pluginutils/src/extractAssignedNames';
+import { extractAssignedNames } from 'rollup-pluginutils';
```

Comment:
We cannot easily do this here because `rollup-pluginutils` itself has a dependency on `micromatch` that cannot be treeshaken. But maybe it is possible to suppress it here via another plugin, I will have a look.

Reply:
Ah no, this cannot happen because micromatch is in fact bundled to make it Node 6 compatible.